components/proxy/src/main/java/com/hotels/styx/admin/AdminServerBuilder.java

```diff
@@ -115,7 +115,9 @@ public HttpServer build() {
         httpRouter.add("/admin/status", new StatusHandler(healthCheckHandler));
         httpRouter.add("/admin/ping", new PingHandler());
         httpRouter.add("/admin/threads", new ThreadsHandler());
-        httpRouter.add("/admin/metrics", new MetricsHandler(environment.metricRegistry(), metricsCacheExpiration));
+        MetricsHandler metricsHandler = new MetricsHandler(environment.metricRegistry(), metricsCacheExpiration);
+        httpRouter.add("/admin/metrics", metricsHandler);
+        httpRouter.add("/admin/metrics/", metricsHandler);
         httpRouter.add("/admin/healthcheck", healthCheckHandler);
         httpRouter.add("/admin/configuration", new StyxConfigurationHandler(staticConfiguration()));
         httpRouter.add("/admin/configuration/origins", new OriginsHandler(backendServicesRegistry));
```

Comment:
Also, this feature would be more useful if it could return all metrics values starting with a prefix. For example:

`curl http://localhost:8080/admin/metrics/jvm.bufferpool`

Would return values for:

```
"jvm.bufferpool.direct.capacity": {
"value": 1
},
"jvm.bufferpool.direct.count": {
"value": 2
},
"jvm.bufferpool.direct.used": {
"value": 2
},
"jvm.bufferpool.mapped.capacity": {
"value": 0
},
"jvm.bufferpool.mapped.count": {
"value": 0
},
"jvm.bufferpool.mapped.used": {
"value": 0
},
```